src/SerilogTracing/Instrumentation/LogEventSpanInstrumentation.cs

```diff
@@ -2,11 +2,11 @@
 using Serilog.Events;
 using SerilogTracing.Core;
 
-namespace SerilogTracing.Interop;
+namespace SerilogTracing.Instrumentation;
 
-static class LogEventSpanExtensions
+static class LogEventSpanInstrumentation
```

Comment:
I would consider avoiding `*Instrumentation` for now, here, since it's not directly tied to any of the instrumentation features. Perhaps something like `LogEventTracingProperties`, if we expect it may eventually carry `IsSpan()`, `TryGetParentId()`, etc.? (Those seem like they'd read acceptably with that name.)